community/event_management/doctype/exhibitor/exhibitor.json

```diff
@@ -36,32 +37,36 @@
    "default": "0",
    "fieldname": "is_paid",
    "fieldtype": "Check",
-   "label": "Is Paid"
+   "label": "is_paid"
   },
   {
    "fieldname": "user",
    "fieldtype": "Link",
    "label": "User",
-   "options": "User"
+   "options": "User",
+   "read_only": 1
```

Comment:
Let's not keep this read-only in the doctype. Our team might add data from the backend. The user field won't be visible to them if it's kept read-only. It should be read-only just in the web forms. Same for the Event field